# Request
geometry_msgs/Pose target_pose
uint16 robot_id
---
# Response
int16 error_code
---
# Feedback
